yices2/src/yices2_term.cpp

```diff
@@ -364,8 +364,32 @@ Sort Yices2Term::get_sort() const
   return Sort(new Yices2Sort(yices_type_of_term(term)));
 }
 
+bool Yices2Term::is_symbol() const
+{
+  // functions are symbols
+  if (is_function)
+  {
+    return true;
+  }
+
+  term_constructor_t tc = yices_term_constructor(term);
+  return (
+          (tc == YICES_UNINTERPRETED_TERM && yices_term_num_children(term) == 0));
+}
+
+bool Yices2Term::is_param() const
+{
+  throw NotImplementedException("Yices2 backend does not support parameters yet.");
+}
+
 bool Yices2Term::is_symbolic_const() const
 {
+  // functions are not constants
+  if (is_function)
+  {
+    return false;
+  }
+
```

Comment:
`return is_symbol();`

Reply:
Addressed in e3e74f36278132d722bf3991c6515fa38de16c1b